app/core/llm_framework/openai_langchain.py

```diff
@@ -1,7 +1,8 @@
 '''Implemetations for lmm_framework interface using langchain'''
 import os
 from typing import List, Tuple
-from langchain.llms import OpenAI
+from langchain.chat_models import ChatOpenAI
```

Comment:
I am curious here. Why the change? What are their differences?

Reply:
I think I was getting an error running it, and it was telling me to make this change. But we could switch it back and see?

Reply:
If tests are running locally then fine